### PR TITLE
🎀 Remove well-known accounts

### LINKED ISSUE #64: Subscribe to the extension accounts list
The application reads accounts from the extension on the page load. However, it is possible to subscribe to the accounts from the extension. This would enable to update the views dynamically without a need to reload the page.

### LINKED ISSUE #1533: Prepare Pioneer for the testnet
In order to prepare the Pioneer for the public release we should make the adjustments:

- remove the well-known tests accounts from the app
- decide whether to use well-known accounts at all (might be cumbersome for local dev but maybe there's a quick way to add them to the extension)

### PR DESCRIPTION
**Pull request by jodator**
Closes #1533

Closes #64 

*   All well-known accounts are removed (can be still used → see the docs)
*   You can hide/unhide accounts on the extension and the app will update accounts lists

### CODE REVIEW

**Comment by jodator** [comment]
@p-sad & @DevNikita16 Please check docs in this PR as well (you need them to test the PR as well :)

**Comment by jodator** [comment]
@thesan FYI mostly. The way how the accounts are loaded will change, so no more automatic dev accounts.

File: docs/README.md [hunk 1]
```diff
@@ -152,3 +162,18 @@ For the `olympia_dev` branch the `defs.json` use this [link](https://github.com/
...
+
+## Using well-known accounts with Polkadot-js extension
+
+Substrate defines [well-known accounts](https://substrate.dev/docs/en/knowledgebase/integrate/subkey#well-known-keys) for `Alice`, `Alice_Stash`, `Bob`, `Bob_Stash`, `Charlie`, `Dave`, `Eve` and `Ferdie`.
+
+To add any of the above:
+
+1. Open extension and click plus sign
+2. Select "Import account from pre-existing seed"
+3. Copy the seed from substrate help page as "existing mnemonic seed"
```
Comment (DevNikita16) [inline docs/README.md:174]:
```suggestion
3. Copy the seed from [substrate help page](https://substrate.dev/docs/en/knowledgebase/integrate/subkey#well-known-keys) as "existing mnemonic seed"
```

**Review by DevNikita16 — APPROVED** [review]
Everything is fine! Just this link could be useful - it took me some time to understand where is the help page.

**Review by p-sad — APPROVED** [review]
LGTM 👍

**Comment by thesan** [comment]
Currently I can't make any transaction on my local network, from the well-known accounts, am I missing a step ?

**Comment by jodator** [comment]
> Currently I can't make any transaction on my local network, from the well-known accounts. Am I missing a step ?

Good catch. It looks like, the well-known key still appears as non-injected (even if injected). The app should open the extension to unlock the key. I was about to create another PR with this but I'm stuck at fixing tests. After my additions, it should be resolved.

**Comment by thesan** [comment]
@jodator I can now transfer JOY from the accounts, but when I try create a membership I get:
```
Uncaught TypeError: memberId is undefined   BuyMembershipModal.tsx:46:72
```

**Comment by jodator** [comment]
> @jodator I can now transfer JOY from the accounts, but when I try create a membership I get:

Ugh.. seems like this bug is also present on main branch as well. I'm merging this PR and see if I can fix that in #1593

**Pull request merged**